### PR TITLE
Feature/login quick start guide/178789849

### PR DESCRIPTION
**Pull request by claireolmstead**
Purpose
---------------
feature: add how-to guide to setting up metamask and torus

Solution
---------------
add instructions text and styling

Change summary
---------------
* added text
* met with heather to discuss styling
* implemented styling
* added conditions for if metamask is setup or not

Steps to Verify
----------------
1. run code
2. make sure that the copy is descriptive enough.
3. try in different browsers
4. try in browser with and without metamask installed

Additional details / screenshot
----------------
- Any suplmenental pictures or material
- Screenshot

### CODE REVIEW

File: src/components/LoginButton.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  loginWalletOptions,
+  loading,
+  onButtonClick,
+}: LoginButtonProps): JSX.Element => {
+  const handleVisibleChange = (visible: boolean) => {
+    setPopoverVisible(visible);
+  };
+
+  const getHeaderLoginButton = (
+    <Popover
```
Comment (claireolmstead) [inline src/components/LoginButton.tsx:25]: Delete popover? UX is confusing to have multiple of the same thing.  lets simplify.

File: package.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
     "kss": "kss --config config/kss-config.json"
   },
   "engines": {
-    "node": ">=14.5.0"
+    "node": "14.5.0"
```
Comment (wilwade) [inline package.json:72]: Was something wrong with `14.16.1`?

File: package.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
     "redux-mock-store": "^1.5.4",
     "resolve": "1.20.0",
     "resolve-url-loader": "3.1.2",
-    "sass-loader": "10.1.1",
+    "sass-loader": "10.2.0",
```
Comment (wilwade) [inline package.json:38]: This small change, but I see 16,316 additions, 53,247 deletions  in the package-lock file. Something feels wrong.

**Review by wilwade — CHANGES_REQUESTED** [review]
The changes look good with a quick glance, but something odd is happening with nodejs and the package-lock file.

File: package.json (hunk 2 above)
Comment (claireolmstead) [reply package.json:72]: It was running into issues with node-sass

File: package.json (hunk 3 above)
Comment (claireolmstead) [reply package.json:38]: This was leftover from me having other node issues and trying to resolve errors.

File: package.json (hunk 2 above)
Comment (claireolmstead) [reply package.json:72]: This was leftover from me having other node issues and trying to resolve errors.  We can roll back.  It works

**Review by wilwade — APPROVED** [review]

**Pull request merged**